lib/common/crmcommon_private.h

```diff
@@ -314,6 +314,7 @@ typedef struct pcmk__ipc_header_s {
     uint32_t size;
     uint32_t flags;
     uint8_t version;
+    uint16_t part_id;               // If this is a multipart message, which part is this?
```

Comment:
Nothing wrong with this and it's probably plenty big, but we don't use `uint16_t` anywhere else. Any reason? Just curious.

Reply:
I *think* the idea was that a GByteArray's maximum size is `guint`, which is the same as `unsigned int`, which is at least 16 bits, hence `uint16_t`.